iree/base/signature_parser.cc

```diff
@@ -14,6 +14,8 @@
 
 #include "iree/base/signature_parser.h"
 
+#include <cstddef>
```

Comment:
And here - header uses `stddef.h` and .cc uses `cstddef` (inverted from logging.h/logging.cc)